code/lightna/magento-os-frontend/css/components/common/search.css

```diff
@@ -1,3 +1,5 @@
+@import "search-suggestions.css";
```

Comment:
will it stay here?

Reply:
no, I will remove all imports in separate pr

Reply:
thou makes no sense to introduce it here to remove later, will do it now